src/app/plan/components/Wizard/WizardComponent.tsx

```diff
@@ -233,6 +278,7 @@ const WizardComponent = (props: IOtherProps & FormikProps<IFormValues>) => {
           targetCluster: props.values.targetCluster,
           selectedStorage: props.values.selectedStorage,
           namespaces: props.values.selectedNamespaces,
+          hookDefinitions: props.values.hookDefinitions
```

Comment:
We should probably add an interface to define the types of this argument object on `addPlanRequest`. Maybe it's just `IFormValues`?

Reply:
In that case, I wonder if we can just pass `props.values` instead of repeating all its properties here

Reply:
removed hookDefinitions as it was unused. We should revisit ts support for addPlan in a later issue.